apps/dotcms-ui-e2e/src/integration/_setup/index.spec.ts

```diff
@@ -0,0 +1,7 @@
+import Utils from '../../support/shared/Utils';
+
+describe('Setup', () => {
+    it('Sets initial DB data', async () => {
+        await Utils.DBSeed();
+    });
+});
```

Comment:
DO we need this?

Reply:
prob not, or will be changed later to be executed on empty starter box

I think when the empty starter brings all the basic data missing right now (container, theme), this won't be needed at all